pkg/apis/rke.cattle.io/v1/zz_generated_deepcopy.go

```diff
@@ -922,6 +922,11 @@ func (in *RKEControlPlaneSpec) DeepCopy() *RKEControlPlaneSpec {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *RKEControlPlaneStatus) DeepCopyInto(out *RKEControlPlaneStatus) {
 	*out = *in
+	if in.AppliedSpec != nil {
+		in, out := &in.AppliedSpec, &out.AppliedSpec
+		*out = new(RKEControlPlaneSpec)
+		(*in).DeepCopyInto(*out)
+	}
```

Comment:
This change is from missing `go generate` from https://github.com/rancher/rancher/pull/40083/files#diff-7396826d3a4d7d1c53716de2bc0c0649150df91d83cbf610ec682163f381e043R39